C0T0/Classes/ApiDataSource/URLRequestBuilder.swift

```diff
@@ -10,20 +10,41 @@ protocol URLRequestBuilder {
 }
 
 class StandardURLRequestBuilder: URLRequestBuilder {
-
+    private enum Constants {
```

Comment:
I must admit I don't like this enum. Will we use `slashes` constant without scheme? If not, U don't see any advantages of keeping separate constants and I would leave just two constants: `http://` and `https://`.